projects/demo/src/app/datepicker/datepicker.demo.scss

```diff
@@ -22,3 +22,59 @@
 .clr-example-input {
   display: inline-block;
 }
+
+.date-range-section {
+  margin: 0 20px;
```

Comment:
This file also needs value tokenization and baselining. 
We discussed the details offline, so should be no problem.

Reply:
It's demo code. My previous comment may be disregarded.